packages/components/form/src/select_field.tsx

```diff
@@ -62,13 +63,14 @@ export const SelectField = function <
     <FormField
       data-testid={`select-field-${inputId}`}
       error={error}
+      hint={hint}
```

Comment:
Consider updating the select field spec to cover this like we do for the text field.

Reply:
Select field hint specs in 5b89b66

Reply:
Select field hint specs in 825f7a5

Reply:
Third time's the charm: Select field hint specs in b14ce84